.github/workflows/push.yml

```
@@ -166,6 +166,12 @@ jobs:
         with:
           RUNS_ON_S3_BUCKET_CACHE: ${{ secrets.S3_DOCKER_CACHE_BUCKET }}
 
+      - name: Install infra modules
+        if: steps.infra-node-modules.outputs.cache-hit != 'true'
```

actionlint (1.7.4)

property "infra-node-modules" is not defined in object type {docker_tags: {conclusion: string; outcome: string; outputs: {string => string}}; git_nx_base: {conclusion: string; outcome: string; outputs: {}}; nodejs_image: {conclusion: string; outcome: string; outputs: {string => string}}; slack: {conclusion: string; outcome: string; outputs: {}}}

(expression)